src/app/_components/StatusBar/utils.tsx

```diff
@@ -11,3 +13,15 @@ export const getColor = (incidentImpact: IncidentImpact) => {
       return 'var(--stacks-colors-status-page-incident-impact)';
   }
 };
+
+export function getIncidentImpactIcon(impact: IncidentImpact) {
```

Comment:
unit test candidate

Reply:
I don't think this is a good candidate, as the it's a if statement with two predicatable outcomes

Reply:
I agree it’s a simple function, but IMO pure functions specially ones that directly affect rendered UI are worth unit testing. It’s low-cost to add and it ensures we won’t regress silently if the logic or enum changes later. Feels like an easy win for coverage and long term confidence. Happy to chat about this further.